src/Sfa.Das.ApprenticeshipInfoService.Api/Controllers/SearchController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Sfa.Das.ApprenticeshipInfoService.Core.Models;
+using Sfa.Das.ApprenticeshipInfoService.Infrastructure.Mapping;
+
+namespace Sfa.Das.ApprenticeshipInfoService.Api.Controllers
+{
+    using System;
+    using System.Net;
+    using System.Web.Http;
+    using Sfa.Das.ApprenticeshipInfoService.Api.Attributes;
+    using Sfa.Das.ApprenticeshipInfoService.Core.Services;
+    using SFA.DAS.NLog.Logger;
+    using Swashbuckle.Swagger.Annotations;
+
+    public class SearchController : ApiController
+    {
+        private readonly IApprenticeshipSearchService _apprenticeshipSearchService;
+        private readonly IProviderSearchService _providerSearchService;
+        private readonly IProviderMapping _providerMapping;
+        private readonly ILog _logger;
+
+        public SearchController(IApprenticeshipSearchService apprenticeshipSearchService, IProviderSearchService providerSearchService, IProviderMapping providerMapping, ILog logger)
+        {
+            _apprenticeshipSearchService = apprenticeshipSearchService;
+            _providerSearchService = providerSearchService;
+            _providerMapping = providerMapping;
+            _logger = logger;
+        }
+
+        /// <summary>
+        /// Search all apprenticeships
+        /// </summary>
+        /// <returns>a search result object</returns>
+        [SwaggerOperation("SearchActiveStandards")]
+        [SwaggerResponse(HttpStatusCode.OK, "OK", typeof(IEnumerable<ApprenticeshipSearchResultsItem>))]
+        [Route("searchApprenticeships/{keywords}/{page}/{take}")]
```

Comment:
Not sure I agree with the page size (take) being a parameter you can define when calling the API. 
Page should default to 1 and not be required on the route